### ISSUE TITLE
Idea add "Notify me when network connection is lost"

### ISSUE THREAD

**Issue by GyulyVGC** [GyulyVGC/sniffnet#632]
### Discussed in https://github.com/GyulyVGC/sniffnet/discussions/622

<div type='discussions-op-text'>

<sup>Originally posted by **markpahulje** September 23, 2024</sup>
Firstly, wanted to say I love this app and great simple design. May I suggest one minor improvement to "Notify me when network connection is lost". Moreover, perhaps a dropped log would be nice as well. I work remote and trying to advise my network provided of time(s) when my connection is dropping and how long it's up for. </div>

**Comment by Berserker-GM** [comment]
Hi, if this is feature is still pending i would like to work on it.

**Comment by GyulyVGC** [comment]
Hey @Berserker-GM thanks for your interest

Yes this is still pending but I'm not really sure whether I want to include this in Sniffnet

A "lost connection" can mean different things and may be due to different factors, and reporting/collecting them could be ambiguous

**Comment by Berserker-GM** [comment]
Yep, sure! Do let me know if you decide to include this. I’ll work on another issue in the meantime.

I can also try to look into whether there’s a reliable way to define and detect a lost connection before attempting an implementation.
